app/views/application/_sub_header.html.erb

```diff
@@ -20,174 +20,176 @@
 	<% active_tab = '' %>
 <% end %>
 
-<nav class="q-navbar-home">
-	<div class="navbar-header">
-		<a class="navbar-brand" href=<%= root_path %>>
-			<img src="/images/quill_header_logo.svg" alt="Quill Logo">
-		</a>
-	</div>
+	<% cache "navbar-#{current_user&.id}-#{user_dropdown_class}" do %>
```

Comment:
Why are we caching based on user id? Is it because we are displaying the username in the navbar?

Reply:
yes.